src/Pos/Txp/Toil/Logic.hs

```diff
@@ -109,55 +76,32 @@ rollbackToil txun = do
 type LocalTxpMode m = ( MonadUtxo m
```

Comment:
Yes, it should be `LocalToilMode`, I forgot to rename when I was doing previous refactoring. Or maybe @pva701 forgot, I don't remember.